IDL/member_id.idl

```diff
@@ -0,0 +1,128 @@
+@mutable
+struct FixId
+{
+    @id(100)
+    octet o;
```

Comment:
Personally, I think it is easier to read if the annotation and the type are in the same line as it is usually written in the specifications. Otherwise, it might be helpful to include a blank line between types. Though this is only a style suggestion and as there is no IDL code style, it can be overlooked.

Reply:
Done in 65376cd